MainModule/Server/Commands/Moderators.lua

```diff
@@ -2503,17 +2503,18 @@ return function(Vargs, env)
 			Prefix = Settings.Prefix;
 			Commands = {"noclip"};
 			Args = {"player"};
-			Description = "NoClips the target player(s); allowing them to walk through walls";
+			Description = "NoClips the target player(s); allowing them to walk through walls.";
 			AdminLevel = "Moderators";
 			Function = function(plr: Player, args: {string})
-				local clipper = Deps.Assets.Clipper:Clone()
-				clipper.Name = "ADONIS_NoClip"
-
 				for i, p in service.GetPlayers(plr, args[1]) do
 					Admin.RunCommand(Settings.Prefix.."clip", p.Name)
-					local new = clipper:Clone()
-					new.Parent = p.Character.Humanoid
-					new.Disabled = false
```

Comment:
i dont mean to be rude but you've legit made all of this bit worse

Reply:
wdym? I don't see a purpose to have an unused clipper script?

Reply:
![image](https://user-images.githubusercontent.com/31691973/202989084-8ef6bef1-abb0-4f1b-b26e-bee0189540e5.png)
you've done this

Reply:
??!

Reply:
wtf

Reply:
You should move a lot of the stuff back in the loop and only preserve it's cloning behavior